src/Compilers/CSharp/Test/Semantic/Semantics/RecordTests.cs

```diff
@@ -4662,6 +4713,166 @@ public void CopyCtor_InaccessibleInMetadata()
                 );
         }
 
+        [Fact, WorkItem(45077, "https://github.com/dotnet/roslyn/issues/45077")]
+        public void CopyCtor_AmbiguitiesInMetadata()
+        {
+            // IL for a minimal `public record B { }` with injected copy constructors
+            var ilSource_template = @"
```

Comment:
>ilSource_template [](start = 16, length = 17)

This constant is used only in one place. Consider inlining, I think this will make the test code easier to follow

Reply:
I think it's helpful to see what we're going to compile (even if contains holes) towards the start of the test. Added comment

---
In reply to: [441036430](https://github.com/dotnet/roslyn/pull/45136#discussion_r441036430) [](ancestors = 441036430)